pkg/test/framework/runtime/components/environment/kube/environment.go

```diff
@@ -92,6 +92,15 @@ func (e *Environment) Scope() lifecycle.Scope {
 	return e.scope
 }
 
+// Is mtls enabled
+func (e *Environment) IsMtlsEnabled() bool {
+	if e.s.Values["global.mtls.enabled"] == "true" {
```

Comment:
This isn't sufficient. It could have been set to true by the values file.

Reply:
Thanks..Fixed